<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Šifra ceste naselja</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QListView" name="SifraCesteNaselja"/>
      </item>
      <item row="1" column="4">
       <widget class="QListView" name="StanjeVozisca"/>
      </item>
      <item row="1" column="2">
       <widget class="QListView" name="VremenskeOkoliscine"/>
      </item>
      <item row="1" column="1">
       <widget class="QListView" name="Lokacija"/>
      </item>
      <item row="1" column="0">
       <widget class="QListView" name="VrstaCeste"/>
      </item>
      <item row="1" column="3">
       <widget class="QListView" name="StanjePrometa"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Vremenske okoliščine</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Lokacija</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Stanje prometa</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QListView" name="SifraOdsekaUlice"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Vrsta ceste</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Stanje vozišča</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QListView" name="VrstaVozisca"/>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Vrsta vozišča</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QListView" name="Mesec"/>
      </item>
      <item row="3" column="3">
       <widget class="QListView" name="Dan"/>
      </item>
      <item row="3" column="5">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="predictBtn">
          <property name="text">
           <string>Predict</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fileBtn">
          <property name="text">
           <string>File Import</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>V naselju</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Dan</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QListView" name="VNaselju"/>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Mesec</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Šifra odseka ulice</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="predictionLbl">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>listView_2</zorder>
   <zorder>listView_6</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>gridLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
